### PR TITLE
Aligned and Improved the Skills section in mobile view

### LINKED ISSUE #391: Aligning and improving the Sharpen your skills section in the mobile view
The Sharpen your skills section in the mobile view is not aligned properly and is creating a problem in the mobile view

I want to make it correct so kindly assign this issue to me @ravikr126 @harikesh409

### PR DESCRIPTION
**Pull request by harshb02**
# Description
Aligned and Improved the Skills section in mobile view

Please include a summary of the change and which issue is fixed.
Fixes #391 

## List any dependencies that are required for this change 
NA

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
On my local host

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
   -
WhatsApp Image 2021-04-19 at 8 32 09 PM

    - After :
    
WhatsApp Image 2021-04-19 at 8 41 23 PM

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ravikr126 — APPROVED** [review]

**Comment by ravikr126** [comment]
REMOVE THE MERGE CONFLICTS @harshb02

**Comment by ravikr126** [comment]
removes conflicts @harshb02

**Comment by harshb02** [comment]
> removes conflicts @harshb02 

Will remove them by tomorrow @ravikr126

File: index.html [hunk 1]
```diff
@@ -146,6 +146,7 @@
 
     <!-- header ends -->
     <!--social media sidebar starts-->
+    <div class="d-none d-lg-block">
```
Comment (harikesh409) [inline index.html:149]: Correct the indentation.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 2]
```diff
@@ -1,5 +1,3 @@
-@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,700;0,900;1,900&display=swap");
```
Comment (harikesh409) [inline css/style.css:1]: Why are the fonts removed? they are still being used in the css file.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**